release notes/v0.47.0.md

```diff
@@ -37,6 +37,16 @@ Thanks, @sapphire-janrain, for contribution!
 
 In some workflows the reflection call should also include some metadata. This PR adds [a new connection parameter `reflectMetadata`](https://k6.io/docs/javascript-api/k6-net-grpc/client/client-connect/#connectparams) that allows to specify the metadata to be sent with the reflection call.
 
+### Improved the browser module's cookie API
+
+The browser module now provides a more complete and robust API for handling cookies. We stabilized the cookie API by defining a new [`Cookie` class](https://k6.io/docs/javascript-api/k6-experimental/browser/browsercontext/cookie) that can be used while creating and retrieving cookies.
+
+We added a new [`browser.cookies([urls])`](https://k6.io/docs/javascript-api/k6-experimental/browser/browsercontext/cookies/) method that returns all cookies from the current [browser context](https://k6.io/docs/javascript-api/k6-experimental/browser/browsercontext). The new API also supports filtering cookies by URL. We also fixed a bug where the `expires` field was not being set correctly while adding cookies using the [`context.addCookie()`](https://k6.io/docs/javascript-api/k6-experimental/browser/browsercontext/addcookies/) method. Lastly, we fixed the existing [`context.clearCookies()`](https://k6.io/docs/javascript-api/k6-experimental/browser/browsercontext/clearcookies) method to clear all cookies from the current [browser context](https://k6.io/docs/javascript-api/k6-experimental/browser/browsercontext). With all these changes, the new API is now more consistent with the [Playwright API](https://playwright.dev/docs/api/class-browsercontext).
```

Comment:
Should we add an example of how to work with the new feature (`browser.cookies([urls])`)?

Reply:
We link to the documentation. I can add an example here if you think it makes sense.

Reply:
Just going by what we've done in the past, it seems that generally we add a short code snippet like what we did for `addCookies` in this [release note](https://github.com/grafana/k6/releases/tag/v0.44.0).

Reply:
WDYT about this example? [a2076be](https://github.com/grafana/k6/pull/3352/commits/a2076be60b1d73aabfdc0145a7c8c56ebb1ca2d5)

Reply:
Thanks!